proposed/cache.md

```diff
@@ -49,7 +49,7 @@ by an integer representing time in seconds, or a DateInterval object.
 typically calculated by adding the TTL to the time when an object is stored, but
 may also be explicitly set with DateTime object.
 
-    An item with a 300 second TTL stored at 1:30:00 will have an expiration at
+    An item with a 300 second TTL stored at 1:30:00 will have an expiration of
```

Comment:
What about:

> An item with a 300 second TTL stored at 1:30:00 will expire at 1:35:00.

Probably doesn't matter too much and if this change hadn't been in this PR it probably wouldn't have even come up so I'm cool w/ leaving it with the new way suggested in this PR as well. :)